packages/roc-package-web-app-react/app/client/render-to-dom.js

```diff
@@ -15,11 +15,23 @@ function renderSync({ renderProps, createComponent, routerRenderFn }, node) {
     ReactDOM.render(finalComponent, node);
 }
 
-export default function renderAsync({ history, routes, ...rest }, node) {
-    match({ history, routes }, (error, redirectLocation, renderProps) => {
+export default function renderAsync(ssr, { history, routes, createComponent, routerRenderFn }, node) {
```

Comment:
We should rename this function since it's not _(always)_ async anymore.

Reply:
You are correct and I will change this. In general I think we should go over the code and optimize it more and refactor it a bit.